.github/workflows/build-test

```diff
@@ -48,21 +48,25 @@ then
     python -m pip install --upgrade mypy
 fi
 for MODULE in $MODULES_TO_TEST; do
-    echo "${MODULE}..."
 
-    cd ${MODULEDIR}/${MODULE}/tests
+    if [[ "${MODULE}" != "pytket-qsharp" || "${PLAT}" != "Windows" ]]
```

Comment:
Can you please raise an issue on Github, and insert a FIXME comment here referencing the issue?

Reply:
Yes

Reply:
https://github.com/microsoft/iqsharp/issues/512

Reply:
I actually meant on _this_ repo. Could you create an issue on pytket-extensions and reference it here as well?

Reply:
Sorry, I misunderstood that. I have addd a new one for pytket-extensions:
https://github.com/CQCL/pytket-extensions/issues/180